pyproject.toml

```diff
@@ -17,21 +17,24 @@ fake-http-header = "^0.3.5"
 [tool.poetry.group.dev.dependencies]
 agentql = "^1.3.0"
 pytest = "^8.3.3"
-pylint = "^3.2.7"
 pytest-mockito = "^0.0.4"
 dill = "*"
-black = "*"
+ruff = "*"
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.black]
-line-length = 100
+[tool.ruff]
+line-length = 120
+target-version = "py39"
 
-[tool.isort]
-profile = "black"
-line_length = 100
-wrap_length = 100
-multi_line_output = 3
-include_trailing_comma = true
+[tool.ruff.format]
+quote-style = "double"
+docstring-code-format = true
+docstring-code-line-length = 20
+
+[tool.ruff.lint]
+select = ["E", "F", "I", "UP"]
+ignore = ["E501"]
```

Comment:
Is it a lot of warnings currently? If not I'd prefer to fix them and not ignore in the future (the lib is small enough)

Reply:
Maybe there is nothing to fix at all. How many these warnings are there?